maxentep/maxent.py

```diff
@@ -248,16 +289,29 @@ def reweight(samples, unbiased_joint, joint):
 
 class HyperMaxentModel(MaxentModel):
     def __init__(self, restraints, prior_model, simulation, reweight=True,
-                 name='hyper-maxent-model', **kwargs):
+                 name='hyper-maxent-model', ref_traj=None, trajs=None, population_fraction=None, ** kwargs):
         super(HyperMaxentModel, self).__init__(
             restraints=restraints, name=name, **kwargs)
         self.prior_model = prior_model
         self.reweight = reweight
         self.unbiased_joint = prior_model(tf.constant([1.]))
+        self.trajs = trajs
         if hasattr(self.unbiased_joint, 'sample'):
             self.unbiased_joint = [self.unbiased_joint]
         self.simulation = simulation
-
+        if ref_traj is not None:
```

Comment:
Just let parent class, MaxEntModel do this. Do not replicate the code. This will be executed when you call super.__init__